docs/source/install-on-debian.rst

```diff
@@ -0,0 +1,370 @@
+===============================================
+Manual install on debian with nginx and certbot
+===============================================
+
+Log in, you need to be able to sudo to root.
+
+Preparations
+============
+
+#. Install nginx:
+
+    .. code-block:: console
+
+        $ sudo apt install nginx
+        .. output ..
+        $
+
+#. Start nginx:
+
+    .. code-block:: console
+
+        $ sudo systemctl start nginx
+        $
+
+#. Visit MYDOMAIN in a browser
+
+    The default nginx page should now be up and running on MYDOMAIN, port 80.
+    Double-check that the page is reachable *worldwide*, that there is no
+    firewall or similar blocking port 80. This is needed for autogenerating the
+    certificate later.
+
+#. Install venv:
+
+    .. code-block:: console
+
+        $ sudo apt install python3-venv
+        .. output ..
+        $
+
+#. Create a user for howitz:
+
+    .. code-block:: console
+
+        $ sudo useradd -m -s /bin/bash howitz
+        $
+
+   This creates a user "howitz" with the group "howitz" which owns the new
+   directory "/home/howitz". The user has (and needs) no password and the shell
+   is set to bash. The "howitz"-user should NOT have root/sudo-access.
+
+#. Become howitz and create a venv:
```

Comment:
Depending on how familiar someone reading this is with Python, the term `venv` might not mean much to many people.  Maybe better to use the term "virtual environment" or "Python virtual environment", and use "venv" as an abbreviation of the term later.